Lib/test/test_sqlite3/test_dump.py

```diff
@@ -41,8 +45,12 @@ def test_table_dump(self):
         [self.cu.execute(s) for s in expected_sqls]
         i = self.cx.iterdump()
         actual_sqls = [s for s in i]
-        expected_sqls = ['BEGIN TRANSACTION;'] + expected_sqls + \
-            ['COMMIT;']
+        expected_sqls = [
+            "PRAGMA foreign_keys=OFF;",
+            "BEGIN TRANSACTION;"
+        ] + expected_sqls + [
```

Comment:
Minor style suggestion.
```suggestion
            "BEGIN TRANSACTION;",
            *expected_sqls,
```